draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -3189,21 +3243,20 @@ http://example.com/high/index.m3u8\n",
    </section>
    <section anchor="cdn-pid" title="CDN Provider ID">
    <t>  
-   The CDN PID consists of the two characters "AS" followed by the CDN
-   provider's Autonomous System number <xref target="RFC1930"/>, then a colon (":") and
-   an additional qualifier that is used to guarantee uniqueness in case 
-   a particular AS has multiple independent CDNs deployed -- for
-   example, "AS64496:0".</t>
+   The CDN PID consists of the two characters "AS" followed by the CDN provider's Autonomous System number 
+   <xref target="RFC1930"/>, then a colon (":") and an additional qualifier that is used to guarantee uniqueness in case 
+   a particular AS has multiple independent CDNs deployed -- for example, "AS64496:0".
+   </t>
 
    <t>  
-   If the CDN provider has multiple ASes, the same AS number SHOULD be
-   used in all messages from that CDN provider, unless there are
-   multiple distinct CDNs.</t>
+   If the CDN provider has multiple ASes, the same AS number SHOULD be used in all messages from that CDN provider, 
+   unless there are multiple distinct CDNs.
+   </t>
 
    <t>  
-   If the CDNI Request Routing Redirection interface (RI) described in
-   <xref target="RFC7975"/> is implemented by dCDN, the CI/T interface and the RI
-   SHOULD use the same CDN PID.</t>
+   If the CDNI Request Routing Redirection interface (RI) described in <xref target="RFC7975"/> is implemented by dCDN, 
```

Comment:
I think the "I" in interface needs to be capitalized here, like in RFC7975.

Reply:
if so, we need to do this across the document

Reply:
Looking back at RFC7975, they also lowercase the "I", so not sure how I saw this as uppercase before--maybe I was tired.  So, I withdraw this comment.